docs/user/tutorial-asgi.rst

```diff
@@ -747,9 +747,8 @@ implementations for production and testing.
     ``self.redis_host``. Such a design might prove helpful for apps that
```

Comment:
This note might need to be updated or even removed altogether if it is no longer needed. How can we hook in `fakeredis` for testing then though? (Monkey-patching or mocking might work too.)

Reply:
with the latest update I made the `redis_from_url` function is still used. I added a wrapper function to `config.py` to change as little as possible. Monkey patching still works.